ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/log/FootstepPlannerLogger.java

```diff
@@ -76,7 +97,7 @@ public class FootstepPlannerLogger
    static final String footstepParametersFileName = "FootstepParametersPacket.json";
```

Comment:
You want to change these to directly match the class names? I got lazy and hardcoded these initially, but it could be something like `static final String footstepParametersFileName = FootstepPlannerParametersPacket.class.getSimpleName();`

Reply:
I don't mind the file names, its what we had before so its easy to keep it